### PR TITLE
Cache plg integ

### PR DESCRIPTION
**Pull request by AntonV1211**
https://doboard.com/1/task/9767

### CODE REVIEW

**Review by svfcode — APPROVED** [review]

File: inc/cleantalk-common.php [hunk 1]
```diff
@@ -833,6 +836,13 @@ function apbct_is_cache_plugins_exists($return_names = false)
     foreach ($constants_of_cache_plugins as $const => $_text) {
         if ( defined($const) ) {
             $out[] = $_text;
+
+            if ($const === 'SPEEDYCACHE_VERSION') {
```
Comment (alexandergull) [inline inc/cleantalk-common.php:840]: This function is used right for detect cache plugins exists, but not for appliyng any changes in the State
 
I believe we can do this here https://github.com/CleanTalk/wordpress-antispam/blob/94afb61b3b8a47178779055663fd7c00d8421554/lib/Cleantalk/ApbctWP/State.php#L603

**Review by alexandergull — CHANGES_REQUESTED** [review]

File: inc/cleantalk-common.php (hunk 1 above)
Comment (AntonV1211) [reply inc/cleantalk-common.php:840]: There will be no information on constants or classes in this place

Comment (alexandergull) [reply inc/cleantalk-common.php:840]: apbct_is_cache_plugins_exists returns names of cache plugins exists if input aprarm is true, maybe compare the name

Comment (AntonV1211) [reply inc/cleantalk-common.php:840]: That's right, but no constants or classes will be found in this place yet, and the function will not return anything

Comment (AntonV1211) [reply inc/cleantalk-common.php:840]: We decided to remove the switch to alt cookies

**Review by alexandergull — APPROVED** [review]

**Pull request merged**